headers/functions/arm9.h

```diff
@@ -98,14 +98,23 @@ void FileSeek(struct file_stream* file, int offset, int whence);
 void FileClose(struct file_stream* file);
 void UnloadFile(void* ptr);
 void LoadFileFromRom(struct iovec* iov, const char* filepath, uint32_t flags);
-bool GetDebugFlag(enum debug_flag flag);
-void SetDebugFlag(enum debug_flag flag, bool val);
+void InitDebug(void);
+void InitDebugFlag1(void);
+bool GetDebugFlag1(enum debug_flag flag);
+void SetDebugFlag1(enum debug_flag flag, bool val);
+void InitDebugStripped6(void);
 int AppendProgPos(char* str, struct prog_pos_info* prog_pos, const char* msg);
+void InitDebugStripped5(void);
 void DebugPrintTrace(const char* msg, struct prog_pos_info* prog_pos);
 void DebugPrint0(const char* fmt, ...);
-bool GetDebugLogFlag(enum debug_log_flag flag);
-void SetDebugLogFlag(enum debug_log_flag flag, bool val);
+void InitDebugFlag2(void);
+bool GetDebugFlag2(enum debug_log_flag flag);
```

Comment:
`enum debug_log_flag` ought to be changed to match the new function name, e.g., `debug_flag_2`

Reply:
Renamed. Also renamed `debug_flag` to `debug_flag_1`.